frontend/src/features/workspace/components/DuplicateFormModal/DupeFormWizardProvider.tsx

```diff
@@ -14,17 +14,18 @@ import {
   CreateFormWizardContextReturn,
 } from '../CreateFormModal/CreateFormWizardContext'
 import { useCommonFormWizardProvider } from '../CreateFormModal/CreateFormWizardProvider'
-import { useWorkspaceRowsContext } from '../WorkspaceFormRow/WorkspaceRowsContext'
 
-export const useDupeFormWizardContext = (): CreateFormWizardContextReturn => {
+export const useDupeFormWizardContext = (
+  formId: string,
+  isTemplate: boolean,
+): CreateFormWizardContextReturn => {
   const { data: dashboardForms, isLoading: isWorkspaceLoading } = useWorkspace()
-  const { activeFormMeta } = useWorkspaceRowsContext()
 
   const { data: previewFormData, isLoading: isPreviewFormLoading } =
     usePreviewForm(
```

Comment:
are we sure that this works as intended if we only call `usePreviewForm` for the template feature? In the backend, `handlePreviewAdminForm` checks for the current user, while `handleGetTemplateForm` does not. So when a different admin tries to dupe the form, this might not work.

Reply:
i cri u are right

Reply:
made the queries somewhat conditional here [db5601b](https://github.com/opengovsg/FormSG/pull/5377/commits/db5601b5a604971618758486ec619167b987d3b7), but not sure if it's the best way to do so, lmk what you think!